extensions/websockets-next/server/runtime/src/main/java/io/quarkus/websockets/next/OpenConnections.java

```diff
@@ -0,0 +1,36 @@
+package io.quarkus.websockets.next;
+
+import java.util.Collection;
+import java.util.stream.Stream;
+
+import io.smallrye.common.annotation.Experimental;
+
+/**
+ * Provides convenient access to all open connections from clients to {@link WebSocket} endpoints on the server.
+ *
+ * @see WebSocketConnection
+ */
+@Experimental("This API is experimental and may change in the future")
+public interface OpenConnections {
+
+    /**
+     *
+     * @return an immutable collection of all open connections
+     */
+    Collection<WebSocketConnection> get();
```

Comment:
Why this method when you already have the `stream()` below that does the same practically?

Reply:
With what java version do we build? If we build with `>= 17`, we can call [`stream().toList()` (`docs.oracle.com`)](https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/util/stream/Stream.html#toList()).